devices/MAX1704x/README.MD

```diff
@@ -0,0 +1,24 @@
+# MAX1704x battery gauge
```

Comment:
Let's be explicit on the supported elements also, you need a new line after a title (MArkdown rule)

```suggestion
# MAX1704x/MAX17043/MAX17044/MAX17048/MAX17049 - Battery gauge

```

Reply:
done